src/libostree/ostree-gpg-verifier.c

```diff
@@ -304,12 +304,85 @@ _ostree_gpg_verifier_add_key_ascii_file (OstreeGpgVerifier *self,
   g_ptr_array_add (self->key_ascii_files, g_strdup (path));
 }
 
+gboolean
+_ostree_gpg_verifier_add_keyfile_path (OstreeGpgVerifier   *self,
+                                       const char          *path,
+                                       GCancellable        *cancellable,
+                                       GError             **error)
+{
+  GError *temp_error = NULL;
+  if (!_ostree_gpg_verifier_add_keyfile_dir_at(self, AT_FDCWD, path,
+                                               cancellable, &temp_error))
+    {
+      if (temp_error)
```

Comment:
We don't really need this check right? (Or alternatively, let's turn it into a `g_assert (temp_error);` ?)

Reply:
`g_assert` sounds good. Addressed in https://github.com/ostreedev/ostree/pull/1773/commits/1e120d2a41107b9e128e1bed2820eceb080a69fb